### ISSUE TITLE
Webtorrent output display not showing up on stream start.

### ISSUE THREAD

**Issue by qascade** [qascade/yast#28]
Make sure that webtorrent output gets redirected to prompt after stream starts as it gives visual assurance to users whether the download started or not.

**Comment by notafk-rajat** [comment]
I would like to work on this issue.

**Comment by qascade** [comment]
Sure. Assigning this to you.

**Comment by qascade** [comment]
@notafk-rajat Hi, I forgot to notice but the template files you did in the previous PR are visible in the repo. Please move them to .workflows folder in this PR.

**Comment by notafk-rajat** [comment]
Okay I'll do it.

**Comment by notafk-rajat** [comment]
Hey @qascade ,  So I was going through readme.md to setup yast on my local environment. I got stuck at step 6 for a while so I had to do `go run main.go` to run the program. So we might need to check into that too.

**Comment by qascade** [comment]
`yast` does not support windows for now. That's why you are unable to set up its config files. I would need to install a linux environment. `yast` has also not been tested on wsl.

**Comment by qascade** [comment]
Would you like to work on bringing support for windows before working on this?? It's an intermediate issue and will teach you file management using Go. We can create two separate issues for the support of WSL and Windows.

**Comment by notafk-rajat** [comment]
Sure I would love to learn something new while trying to fix this issue. That is the main reason I want to do open-source, learn on the go! (no referring to golang).
I have WSL installed I'll try on it and let you know what I observe there.

**Comment by notafk-rajat** [comment]
Also, I think it would be better to exclude the moving of template files to workflow from this issue.

**Comment by qascade** [comment]
I asked someone else to move the templates, so you don't need to do that anymore. Anyways, thanks for your continued interest in my project. If you have any questions or blockers do let me know.

**Comment by qascade** [comment]
@notafk-rajat As almost all of the support issues are resolved, you can start looking back into this issue.

**Comment by notafk-rajat** [comment]
Well, I too have the same issue, I don't get the output on stream start (as I said previously in #51  and #53 ), because the WSL
and windows cannot support `yast` currently.

So I'm not sure what to do here.

**Comment by qascade** [comment]
Ok, the only solution for you is to get linux environment. You try getting a VM. But if that is not feasible for you, I'm afraid you won't be able to contribute to yast.

**Comment by notafk-rajat** [comment]
I tried VM once but it was kinda laggy that's why I switched over to WSL. I'll try VM again and let you know. But it'll take a while maybe the day after tomorrow cause I have exams going on.

**Comment by notafk-rajat** [comment]
In the meantime can you explain the issue in a bit more detail? Like on starting the stream the output screen is not popping up? The media player window?

**Comment by qascade** [comment]
Use webtorrent-cli separately and run the stream command by hardcoding magnet. You will see an output on screen but that is not happening when the same command is executed in golang. Basically we need to explicitly pipe the `cmd` go-routine's output to `os.stdout`

**Comment by notafk-rajat** [comment]
Hmm great explanation, but I'll need to dumb it down a little for myself now. Cause the words like 'hardcoding magnet' went above my head. I know about how there are magnet links to download torrents.

**Comment by qascade** [comment]
Hardcoding means just explicitely copy pasting.

**Comment by qascade** [comment]
See the code where WebTorrent CLI command is getting executed 
. The function name is `StartStream()` if I remember correctly.

**Comment by notafk-rajat** [comment]
Hey @qascade, you can remove me from this issue. I don't think I'll be able to work on yast unless I dual boot my system with Linux.
